README.md

```diff
@@ -20,6 +20,9 @@ It's very opinionated out the box (as software should be!) but allows you to cus
 - `pkgs`: Optional additional Nix packages to install in the environment.
 - `apt`: Optional additional Apt packages to install in the environment.
 - `push`: A boolean flag to indicate whether to push the built image to the registry. Default is `false`. Required for multi-architecture builds.
+- `cache`: A boolean flag to indicate whether to use the build cache. Default is `false`. 
+  (NOTE: The cache is shared between all builds in the repository. Some cache metadata will be inlined in the final image.)
+- `cache_tag`: A single tag to use for the cache image. Required if `cache` is `true`.
```

Comment:
Can you give an example?

Reply:
I added a default value note, should be sufficient as an example